app/components/Settings/Settings.react.js

```diff
@@ -510,7 +510,7 @@ class Settings extends Component {
                                 <p style={{textAlign: 'right'}}>
                                     Not logged in
                                     <br/>
-                                    <a onClick={() => window.location.assign('/login')}>Log into Plotly</a>
+                                    <a href="/login">Log into Plotly</a>
```

Comment:
If you want to make this change, you also need to change the `<a>` above like this:

```diff
user@host:~/github/plotly-database-connector$ git diff
diff --git a/app/components/Settings/Settings.react.js b/app/components/Settings/Settings.react.js
index d46f1f8..ab0c608 100644
--- a/app/components/Settings/Settings.react.js
+++ b/app/components/Settings/Settings.react.js
@@ -504,7 +504,7 @@ class Settings extends Component {
                                 <p style={{textAlign: 'right'}}>
                                     {`Logged in as "${username}"`}
                                     <br/>
-                                    <a onClick={logout}>Log Out</a>
+                                    <a href="/" onClick={logout}>Log Out</a>
                                 </p>
                             ) : (
                                 <p style={{textAlign: 'right'}}>
```

Otherwise users will be redirected to the login page immediately after logging out:

![peek 2018-07-25 15-59](https://user-images.githubusercontent.com/6199391/43209118-cc1acba0-9023-11e8-8651-c2a68d6f513c.gif)

Reply:
I am just reverting this. Not a necessary change. Thanks 👍